css/dialog.css

```diff
@@ -0,0 +1,75 @@
+.item-dialog {
+    width: 95%;
+    margin: auto auto;
+    background-color: var(--primary-light);
+    border: none;
+    padding: var(--size-4);
+}
+
+.item-dialog::backdrop {
+    background-color: rgba(0, 0, 0, 0.301);
+}
+
+.item-dialog-form {
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.item-dialog-form h3 {
+    margin: var(--size-3) 0;
+}
+
+.item-dialog-form label {
+    font-size: var(--font-size-small);
+}
+
+.item-dialog-form input {
+    padding-left: var(--size-1);
+}
+
+.item-dialog-form input,
+.item-dialog-form select {
+    width: 100%;
+    height: var(--size-4);
```

Comment:
I think variables used here are too generic for where they're used. Try having tokens such as:

```
--field-font-size: var(--font-size-small);
--field-height: var(--size-4);
--field-spacing: var(--size-4);
```